src/main/java/io/kestra/plugin/odoo/package-info.java

```diff
@@ -0,0 +1,8 @@
+@PluginSubGroup(
+    title = "Odoo",
+    description = "Integrate with Odoo ERP system via XML-RPC API for data orchestration and workflow automation.",
+    categories = PluginSubGroup.PluginCategory.DATABASE
```

Comment:
Odoo is an ERP (so a tool), not a database, even if it uses PostgreSQL behind the scenes.

```suggestion
    categories = PluginSubGroup.PluginCategory.TOOL
```